Gemfile

```diff
@@ -1,20 +1,17 @@
 source "https://rubygems.org"
 
-gem "sinatra"
-gem "sinatra-activerecord"
+gem 'actionmailer'
 gem 'pg'
+gem 'sinatra'
+gem 'sinatra-activerecord'
 gem 'sinatra-flash'
-gem 'actionmailer'
-
-gem 'pry'
-gem 'shotgun'  #Autoreloader: $shotgun server.rb
-gem "capybara"
-gem "rspec"
 gem 'sinatra-contrib'
-gem 'database_cleaner'
-gem 'poltergeist'
-
-
-
-
 
+group :development, :test do
+  gem 'capybara'
+  gem 'database_cleaner'
+  gem 'poltergeist'
+  gem 'pry'
+  gem 'rspec'
+  gem 'shotgun'  #Autoreloader: $shotgun server.rb
+end
```

Comment:
These gems aren't needed in `production`, so limit their environments.. (And alphabetize, to make it easier to scan)